src/inc/stackstring.h

```diff
@@ -0,0 +1,196 @@
+// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license. 
+// See the LICENSE file in the project root for more information. 
+
+
+#ifndef __STACKSTRING_H_
```

Comment:
> Including SString derived types in the Wrapper APIs was causing a lot of trouble

What kind of trouble are you referring to? Your changes are in the VM and VM uses SString already, so the wrapper implementation should be able to do the same as well. Or is there some context missing?